src/env/node.js

```diff
@@ -41,11 +48,12 @@ function getTree (msg, i) {
 	if (msg.collapsed !== undefined) {
 		let {collapsed, highlighted, children} = msg;
 
-		let icon = msg.collapsed ? "▶︎ " : "▼ ";
-		msg = icon + msg;
+		let icon = collapsed ? "▷" : "▽";
 		if (highlighted) {
+			icon = `<c green><b>${ collapsed ? "▶︎" : "▼" }</b></c>`;
```

Comment:
These should not be magic strings randomly in the code, they should be referenced from some kind of data structure.